functions/v2/firebase/FirebaseFirestore/Function.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START functions_cloudevent_firebase_firestore]
+using CloudNative.CloudEvents;
+using Google.Cloud.Functions.Framework;
+using Google.Events.Protobuf.Cloud.Firestore.V1;
+using Google.Protobuf.WellKnownTypes;
+using Microsoft.Extensions.Logging;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace FirebaseFirestore
+{
+    public class Function : ICloudEventFunction
```

Comment:
When 1.3.0 is out, you shouldn't have to do this. You should be able to use `ICloudEventFunction<DocumentEventData>`, even with binary data.

Reply:
Updated version and tested. Everything worked as expected. Reducing PR to just upgrading to version 1.3.0 on existing sample